DB/init.sql

```diff
@@ -0,0 +1,56 @@
+
+DROP TABLE Users;
+DROP TABLE Courses;
+DROP TABLE Tasks;
+DROP TABLE Subjects;
+DROP TABLE IsIn;
+DROP TABLE HasEvaluation;
+
+CREATE TABLE Users
+(
+    Id          INTEGER PRIMARY KEY,
+    LoginName   VARCHAR(32),
+    Vorname     VARCHAR(16),
+    Name        VARCHAR(16),
+    Type        INTEGER(1),
```

Comment:
Type ist die Rolle, oder? Wäre da nicht ein VARCHAR(10) besser wo man die rolle (student, teacher, admin) direkt reinschreibt? dann braucht man nicht irgendwo eine aufschlüsselng mit 1=schüler, 2= lehrer usw. ist denk ich intuitiver, auch wenn man das im Frontend verwendet

Reply:
Ja, mein Gedanke war natürlich, etwas Speicherplatz zu sichern, weil dann natürlich für jeden Nutzer ein ganzer String in der DB steht, aber fürs Erste ist die Verständlichkeit wohl wichtiger. Ich ändere es.